language/pl-PL/com_users.ini

```
@@ -4,119 +4,159 @@
+COM_USERS_EMAIL_REGISTERED_WITH_ADMIN_ACTIVATION_BODY="Witaj {NAME},\n\nDziękujemy za rejestrację na {SITENAME}. Twoje konto zostało już utworzone, ale zanim będzie można z niego korzystać, wymaga weryfikacji.\nW tym celu naciśnij poniższy odnośnik albo skopiuj go i wklej w pasku adresu Twojej przeglądarki:\n{ACTIVATE} \n\nNa tej podstawie administrator witryny zostanie poproszony o zatwierdzenie Twojego konta.\nO zatwierdzeniu konta otrzymasz powiadomienie.\nGdy Twoje konto zostanie zatwierdzone, będzie możliwe logowanie się na stronie{SITEURL}, przy pomocy poniższej nazwy użytkownika i hasła:\n\nNazwa użytkownika: {USERNAME}\nHasło: {PASSWORD_CLEAR}"
+COM_USERS_EMAIL_REGISTERED_WITH_ADMIN_ACTIVATION_BODY_NOPW="Witaj {NAME},\n\nDziękujemy za rejestrację na {SITENAME}. Twoje konto zostało już utworzone, ale zanim będzie można z niego korzystać, wymaga uaktywnienia.\n\nW tym celu naciśnij poniższy odnośnik albo skopiuj go i wklej w pasku adresu Twojej przeglądarki:\n {ACTIVATE}\n\nNa tej podstawie administrator witryny zostanie poproszony o zatwierdzenie Twojego konta.\n\nO zatwierdzeniu konta otrzymasz powiadomienie.\n\nGdy Twoje konto zostanie zatwierdzone, będzie możliwe logowanie się na stronie {SITEURL}, przy pomocy poniższej nazwy użytkownika:\n\nNazwa użytkownika: {USERNAME}"
+COM_USERS_EMAIL_USERNAME_REMINDER_BODY="Witaj,\n\nPrzypominamy na Twoją prośbę nazwę użytkownika dla Twojego konta na {SITENAME}.\n\nTwoja nazwa użytkownika to: {USERNAME}.\n\nJeśli chcesz zalogować się teraz na swoje konto, wybierz poniższy odnośnik.\n\n{LINK_TEXT} \n\nDziękujemy!"
+COM_USERS_EMAIL_USERNAME_REMINDER_SUBJECT="Twoja nazwa użytkownika na {SITENAME}"
 COM_USERS_FIELD_PASSWORD_RESET_LABEL="Adres e-mail"
 COM_USERS_FIELD_REMIND_EMAIL_LABEL="Adres e-mail"
 COM_USERS_FIELD_RESET_CONFIRM_TOKEN_LABEL="Kod weryfikacyjny"
-COM_USERS_FIELD_RESET_CONFIRM_USERNAME_LABEL="Użytkownik"
+COM_USERS_FIELD_RESET_CONFIRM_USERNAME_LABEL="Nazwa użytkownika"
 COM_USERS_FIELD_RESET_PASSWORD1_LABEL="Hasło"
-COM_USERS_FIELD_RESET_PASSWORD1_MESSAGE="Podane hasła nie są identyczne. Wpisz żądane hasło w polu Hasło oraz potwierdź je w polu Powtórz hasło."
-COM_USERS_FIELD_RESET_PASSWORD2_LABEL="Powtórz hasło administratora"
+COM_USERS_FIELD_RESET_PASSWORD1_MESSAGE="Podane hasła nie są identyczne. Wprowadź żądane hasło w polu hasła oraz potwierdź je w polu powtórzenia hasła"
+COM_USERS_FIELD_RESET_PASSWORD2_LABEL="Powtórz hasło"
 COM_USERS_INVALID_EMAIL="Błędny adres e-mail"
+COM_USERS_LBL_SELECT_INSTRUCTIONS="Wybierz, w jaki sposób chcesz zweryfikować swój login do tej strony internetowej."
 COM_USERS_LOGIN_DEFAULT_LABEL="Login użytkownika"
 COM_USERS_LOGIN_REGISTER="Nie masz konta?"
-COM_USERS_LOGIN_REMEMBER_ME="Pamiętaj mnie"
-COM_USERS_LOGIN_REMIND="Nie pamiętasz nazwy?"
+COM_USERS_LOGIN_REMEMBER_ME="Zapamiętaj mnie"
+COM_USERS_LOGIN_REMIND="Nie pamiętasz nazwy użytkownika?"
```

W kontekście jest zrozumiałe, o co chodzi. W module, zwłaszcza na urządzeniach mobilnych, etykieta łącza trochę przydługa. Pozostawiłbym bez zmian (ewentualnie dodać krótsze "swojej nazwy"). 
```suggestion
COM_USERS_LOGIN_REMIND="Nie pamiętasz nazwy?"
```